common/src/main/resources/db/changelog/v001/ab2d-777-resolve_diff_ids.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE opt_out ADD COLUMN mbi VARCHAR(255);
```

Comment:
liquibase changelogs using sql format may need to begin with 

```
--liquibase formatted sql
```
and every change-set needs to begin with a line that looks something like this:

```
--changeset <author_id>:<change_set_id> failOnError:true
```
This is used by liquibase during the database migration process.

https://www.liquibase.org/documentation/sql_format.html